static/src/js/components/Facets/Facets.js

```diff
@@ -58,39 +58,46 @@ const Facets = (props) => {
   const keywordsFacet = {
     ...cmrFacetDefaults,
     title: 'Keywords',
+    autocompleteType: 'science-keywords',
```

Comment:
I feel like this is generally an underscore, I know CMR 'offers' both but we currently use `science_keywords_h` for facets, is this our own internal key or something were asking from CMR? Maybe should be included in [CMR-6287](https://bugs.earthdata.nasa.gov/browse/CMR-6287)?

EDIT: Looks like the other autocompleteTypes below are using camel case -- we should match those.